src/geometry/GeoTrackView.hh

```diff
@@ -48,15 +48,39 @@ class GeoTrackView
     // Move to the next boundary
     inline CELER_FUNCTION void move_next_step();
 
+    // Update current volume, called whenever move reaches boundary
+    inline CELER_FUNCTION void move_next_volume();
+
+    // A tiny push to make sure tracks go over boundaries, to minimize chances of getting stuck at boundaries
+    CELER_CONSTEXPR_FUNCTION real_type tolerance()
```

Comment:
I think this should be a `static` function, not a mutable member function.

Reply:
Done.